src/client/results/Feed.js

```diff
@@ -1,24 +1,34 @@
 import React, { Component } from 'react';
 import moment from 'moment';
 
-const Feed = ({ feed, hiddenWords, hiddenUsers }) => {
-  const filteredFeed = filterPostsForFeed(feed, hiddenWords, hiddenUsers);
+class Feed extends Component {
+  componentDidUpdate() {
+    $('.popup').popup();
+  }
 
-  return (
-    <div>
-      <div className="ui two column grid">
-        <div className="column">
-          <h3>Search results</h3>
-        </div>
-        <div className="right aligned column">
-          Showing {filteredFeed.length}/{feed.length} posts
-        </div>
-      </div>
-      <div className="ui divided items">
-        {filteredFeed.map((feedItem) => (<FeedItem content={feedItem} />))}
-      </div>
-    </div>
-  );
+  render() {
+    const feed = this.props.feed;
+    const hiddenWords = this.props.hiddenWords;
+    const hiddenUsers = this.props.hiddenUsers;
+
+    const filteredFeed = filterPostsForFeed(feed, hiddenWords, hiddenUsers);
+
+    return (
+      <div>
+        <div className="ui two column grid">
+          <div className="column">
+            <h3>Search results</h3>
+          </div>
+          <div className="right aligned column">
+            Showing {filteredFeed.length}/{feed.length} posts
+          </div>
+        </div>
+        <div className="ui divided items">
+          {filteredFeed.map((feedItem) => (<FeedItem content={feedItem}/>))}
+        </div>
+      </div>
+    );
+  }
 };

```

Comment:
worth adding proptypes to this ? It's up to you, just thought may as well make you aware of it X)  

https://facebook.github.io/react/docs/reusable-components.html